iOS build issue - include of non-modular header inside framework module

I'm unable to build my project after updating to `0.9.1+1`.

I get the following error:

```
 In file included from /myproject/build/ios/Release-stage-iphoneos/flutter_appauth/flutter_appauth.framework/Headers/flutter_appauth-umbrella.h:14:
    /myproject/build/ios/Release-stage-iphoneos/flutter_appauth/flutter_appauth.framework/Headers/OIDExternalUserAgentIOSNoSSO.h:20:9: error: include of non-modular header inside framework module 'flutter_appauth.OIDExternalUserAgentIOSNoSSO': '/myproject/build/ios/Release-stage-iphoneos/AppAuth/AppAuth.framework/Headers/OIDExternalUserAgent.h' [-Werror,-Wnon-modular-include-in-framework-module]
    #import "OIDExternalUserAgent.h"
            ^
    1 error generated.
    /myproject/ios/Runner/GeneratedPluginRegistrant.m:62:9: fatal error: could not build module 'flutter_appauth'
    #import <flutter_appauth/FlutterAppauthPlugin.h>
     ~~~~~~~^
    2 errors generated.
```

Any ideas on how to fix it?

--- a/flutter_appauth/CHANGELOG.md
+++ b/flutter_appauth/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 0.9.2
+
+* [iOS] Fixes release build issue  about including a non-modular header that came about from release 0.9.1+1
+
 ## 0.9.1+1
 
 * Fixes warnings picked up in Xcode raised in issue [123](https://github.com/MaikuB/flutter_appauth/issues/123)

--- a/flutter_appauth/CHANGELOG.md
+++ b/flutter_appauth/CHANGELOG.md
@@ -1,6 +1,6 @@
 ## 0.9.2
 
-* [iOS] Fixes release build issue  about including a non-modular header that came about from release 0.9.1+1
+* [iOS] Fixes release build issue about including a non-modular header that came about from release 0.9.1+1. If you experienced issues with 0.9.1+1 (in my experience only occurs if your iOS app uses Swift), please update to 0.9.2
 
 ## 0.9.1+1
 

--- a/flutter_appauth/ios/Classes/FlutterAppauthPlugin.h
+++ b/flutter_appauth/ios/Classes/FlutterAppauthPlugin.h
@@ -1,6 +1,5 @@
 #import <Flutter/Flutter.h>
-
-@protocol OIDExternalUserAgentSession;
+#import <AppAuth/AppAuth.h>
 
 @interface FlutterAppauthPlugin : NSObject<FlutterPlugin>
 

--- a/flutter_appauth/ios/Classes/FlutterAppauthPlugin.m
+++ b/flutter_appauth/ios/Classes/FlutterAppauthPlugin.m
@@ -1,7 +1,6 @@
 #import "FlutterAppauthPlugin.h"
-#import "AppAuth.h"
-
 #import "OIDExternalUserAgentIOSNoSSO.h"
+
 @interface ArgumentProcessor : NSObject
 + (id _Nullable)processArgumentValue:(NSDictionary *)arguments withKey:(NSString *)key;
 @end

--- a/flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.h
+++ b/flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.h
@@ -17,7 +17,7 @@
  */
 
 #import <UIKit/UIKit.h>
-#import "OIDExternalUserAgent.h"
+#import <AppAuth/AppAuth.h>
 
 @class SFSafariViewController;
 

--- a/flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.m
+++ b/flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.m
@@ -9,9 +9,6 @@
 #import <SafariServices/SafariServices.h>
 #import <AuthenticationServices/AuthenticationServices.h>
 
-#import "OIDErrorUtilities.h"
-#import "OIDExternalUserAgentSession.h"
-#import "OIDExternalUserAgentRequest.h"
 
 #if !TARGET_OS_MACCATALYST
 

--- a/flutter_appauth/pubspec.yaml
+++ b/flutter_appauth/pubspec.yaml
@@ -1,6 +1,6 @@
 name: flutter_appauth
 description: This plugin provides an abstraction around the Android and iOS AppAuth SDKs so it can be used to communicate with OAuth 2.0 and OpenID Connect providers
-version: 0.9.1+1
+version: 0.9.2
 homepage: https://github.com/MaikuB/flutter_appauth/tree/master/flutter_appauth
 
 environment:

